share/spack/gitlab/cloud_pipelines/stacks/hep/spack.yaml

```
@@ -0,0 +1,64 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: when_possible
+
+  packages:
+    all:
+      require: '%gcc target=x86_64_v3'
+      providers:
+        blas: [openblas]
+        mpi: [mpich]
+        tbb: [intel-tbb]
+      variants: +mpi
+
+    # Mark geant4 data as external to prevent wasting bandwidth on GB-scale files
+    geant4-data:
+      externals:
+        - spec: geant4-data@11.3.0
+          prefix: /usr
+        - spec: geant4-data@11.2.2
+          prefix: /usr
+        - spec: geant4-data@11.2.0
+          prefix: /usr
+        - spec: geant4-data@11.1.0
+          prefix: /usr
+        - spec: geant4-data@11.0.0
+          prefix: /usr
+
+  specs:
+  # CPU
+  - acts +analysis +dd4hep +edm4hep +examples +fatras +geant4 +hepmc3 +podio +pythia8 +python +tgeo
```

To avoid defaulting to `cxxstd=17` (and `acts@:35`) we explicitly require `cxxstd=20`:
```suggestion
  - acts +analysis +dd4hep +edm4hep +examples +fatras +geant4 +hepmc3 +podio +pythia8 +python +tgeo cxxstd=20
```